modules/ocf_desktop/files/xsession/fix-displays

```diff
@@ -31,26 +44,20 @@ if __name__ == '__main__':
     # |    \------+ \-------/    \------+    \------+  |
     # +------------------------------------------------+
 
+    daisy_chained = is_daisy_chaining(monitors)
+
+    if daisy_chained:
+        sorted_monitors = list(sorted(monitors, reverse=True))
+    else:
+        sorted_monitors = list(sorted(monitors))
+
     prev = None
-    for monitor in sorted(monitors):
+    for monitor in sorted_monitors:
         if prev:
             subprocess.check_call(['xrandr', '--output', monitor, '--right-of', prev])
         prev = monitor
 
-    # special case for eruption: middle monitor is rotated left 90 degrees
-    if len(monitors) == 3 and hostname == 'eruption':
-        # left/right monitors are moved down a bit; move middle up to compensate
-        middle_offset = -305
-        right_offset = 320
-
-        # need to get the current x position of this monitor first
-        line = next(filter(lambda line: monitors[1] in line,
-                           subprocess.check_output(['xrandr']).decode('utf8').splitlines()))
-        x_pos = re.search('\\b\\d+x\\d+\\+(\\d+)\\+', line).group(1)
-
-        subprocess.check_call(['xrandr', '--output', monitors[1],
-                               '--rotate', 'left', '--pos', '{}x{}'.format(x_pos, middle_offset)])
-
-        x_pos = str(int(x_pos) + 1200)
-        subprocess.check_call(['xrandr', '--output', monitors[2],
-                               '--pos', '{}x{}'.format(x_pos, right_offset)])
+    if daisy_chained:
+        # Disable power saving, this is necessary for some reason
+        subprocess.check_call(['xset', 'dpms', 'force', 'off'])
+        subprocess.check_call(['xset', 'dpms', 'force', 'on'])
```

Comment:
Perhaps we should call out to the new `fix-daisy` script here.

Reply:
Yeah, that works!